cfme/tests/containers/test_basic_metrics.py

```diff
@@ -1,13 +1,41 @@
 import pytest
 import requests
 from cfme.containers.provider import ContainersProvider
-
+from cfme.utils.appliance.implementations.ui import navigate_to
 
 pytestmark = [
     pytest.mark.usefixtures('setup_provider'),
     pytest.mark.tier(1),
     pytest.mark.provider([ContainersProvider], scope='function')]
 
+
+METRICS_CAPTURE_THRESHOLD_IN_MINUTES = 5
+ROLLUP_METRICS_CALC_THRESHOLD_IN_MINUTES = 50
+
+
+@pytest.fixture(scope="module")
+def reduce_metrics_collection_threshold(appliance):
+    appliance.ssh_client.put_file("scripts/openshift/change_metrics_collection_threshold.rb",
+                                  "/var/www/miq/vmdb")
+    appliance.ssh_client.run_rails_command(
+        "change_metrics_collection_threshold.rb {threshold}.minutes".format(
+            threshold=METRICS_CAPTURE_THRESHOLD_IN_MINUTES))
+
+
+@pytest.fixture(scope="module")
+def enable_capacity_and_utilization(appliance):
+    appliance.server.settings.enable_server_roles('ems_metrics_coordinator',
```

Comment:
Required: please use yield fixture to set server roles to default after the test is done

Reply:
done